backend/core/entities/file_converter.py

```diff
@@ -12,23 +12,31 @@ def __init__(self, file, protected_attributes: list[str]):
 
     # refactor clean_dataset to use protected_attributes to clean dataset
     def clean_dataset(self):
-        priv_gender = self.find_priv( "sender_gender")
-        priv_race = self.find_priv("sender_race")
+        priv_sender_gender = self.find_priv("sender_gender")
```

Comment:
ooo wait one thing - can we refactor it so that clean_dataset acts accordingly using whatever is in the protected_attributes?